src/GameLogic/PlayerActions/Craftings/FenrirUpgradeCrafting.cs

```diff
@@ -82,22 +81,22 @@ protected override int GetPrice(byte successRate, IList<CraftingRequiredItemLink
         if (randomWeapons.Any())
         {
             items.Add(new CraftingRequiredItemLink(randomWeapons, new TransientItemCraftingRequiredItem { MinimumAmount = 1, MaximumAmount = 1, Reference = 2 }));
-            successRateByItems = (byte)Math.Min(79, randomWeapons.Sum(this._priceCalculator.CalculateBuyingPrice) * 100 / 1_000_000);
+            successRateByItems = (byte)Math.Min(79, randomWeapons.Sum(this._priceCalculator.CalculateSellingPrice) * 100 / 1_000_000);
```

Comment:
Sources I checked do say it is the buying price, but the client values are based on the selling price.